stable/datadog/templates/daemonset.yaml

```diff
@@ -83,8 +84,8 @@ spec:
       - name: config
         emptyDir: {}
       - hostPath:
-          path: {{ template "datadog.dockerOrCriSocketPath" . }}
-        name: runtimesocket
+          path: {{ dir (include "datadog.dockerOrCriSocketPath" .) }}
```

Comment:
Not sure what dir() does but I'm guessing it does not work for Windows named pipes

Reply:
Change reverted for Windows to that it applies to Linux only.